modules/govuk/manifests/node/s_graphite.pp

```diff
@@ -91,41 +91,74 @@
     extra_config => $cors_headers,
   }
 
-  ## Make compressed archives of Whisper data to backup off-box rather than
-  ## backing up the raw files
-  file { '/usr/local/bin/govuk_backup_graphite_whisper_files':
-    ensure => present,
-    source => 'puppet:///modules/govuk/usr/local/bin/govuk_backup_graphite_whisper_files',
-    mode   => '0755',
-  }
-
-  file { '/usr/local/bin/govuk_delete_ephemeral_interface_data':
-    ensure => present,
-    source => 'puppet:///modules/govuk/usr/local/bin/govuk_delete_ephemeral_interface_data',
-    mode   => '0755',
-  }
-
-  package { 'pigz':
-    ensure => installed,
-  }
-
-  file { '/opt/graphite/storage/backups':
-    ensure => 'directory',
-    owner  => 'govuk-backup',
-    group  => 'govuk-backup',
-    mode   => '0770',
-  }
-
-  cron::crondotdee { 'create_compressed_archive_of_whisper_data':
-    command => '/usr/local/bin/govuk_backup_graphite_whisper_files',
-    hour    => $graphite_backup_hour,
-    minute  => 45,
-  }
+  ## Backing up whisper database directly to S3 using the whisper-backup script
+  if $::aws_migration {
+    $apt_mirror_hostname  = undef
+    apt::source { 'whisper-backup':
+      ensure       => present,
+      location     => "http://${apt_mirror_hostname}/whisper-backup",
```

Comment:
Because of `$apt_mirror_hostname  = undef` this ends up being `http:///whisper-backup` and we get the following errors when doing an `apt-get update`:

```
W: Failed to fetch http:/whisper-backup/dists/trusty/Release.gpg  Unable to connect to :http:

W: Failed to fetch http:/whisper-backup/dists/trusty/main/binary-amd64/Packages  Unable to connect to :http:

W: Failed to fetch http:/whisper-backup/dists/trusty/main/i18n/Translation-en_GB  Unable to connect to :http:

W: Failed to fetch http:/whisper-backup/dists/trusty/main/i18n/Translation-en  Unable to connect to :http:

E: Some index files failed to download. They have been ignored, or old ones used instead.
```